### ISSUE TITLE
[cli] `polymer serve/lint --help` is broken

### ISSUE THREAD

**Issue by jsilvermist** [Polymer/tools#541]
This worked in 1.7.2, unknown for 1.7.3, broken in 1.7.4 and 1.7.6.

All other `--help`'s work, but `polymer serve --help` errors with:

> WSL Ubuntu
```bash
$ polymer serve --help
error: [cli.main]   cli runtime exception: Error: Found extraneous } in Chalk template literal
error: [cli.main]   Error: Found extraneous } in Chalk template literal
    at tmp.replace (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/templates.js:109:11)
    at String.replace (<anonymous>)
    at module.exports (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/templates.js:99:6)
    at chalkTag (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/index.js:221:9)
    at Chalk.chalk.template (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/index.js:36:20)
    at format (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/chalk-format.js:4:10)
    at chalkFormat (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/chalk-format.js:10:12)
    at OptionList.definitions.map.def (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/option-list.js:41:22)
    at Array.map (<anonymous>)
    at new OptionList (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/option-list.js:38:30)
```
> Command Prompt
```cmd
C:\Users\Jason>polymer serve --help
error: [cli.main]   cli runtime exception: Error: Found extraneous } in Chalk template literal
error: [cli.main]   Error: Found extraneous } in Chalk template literal
    at tmp.replace (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\node_modules\chalk\templates.js:109:11)
    at String.replace (<anonymous>)
    at module.exports (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\node_modules\chalk\templates.js:99:6)
    at chalkTag (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\node_modules\chalk\index.js:221:9)
    at Chalk.chalk.template (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\node_modules\chalk\index.js:36:20)
    at format (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\lib\chalk-format.js:4:10)
    at chalkFormat (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\lib\chalk-format.js:10:12)
    at OptionList.definitions.map.def (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\lib\option-list.js:41:22)
    at Array.map (<anonymous>)
    at new OptionList (C:\Users\Jason\AppData\Roaming\npm\node_modules\polymer-cli\node_modules\command-line-usage\lib\option-list.js:38:30)
```

EDIT: `polymer lint --help` also errors:
> WSL Ubuntu
```bash
$ polymer lint --help
error: [cli.main]   cli runtime exception: Error: Found extraneous } in Chalk template literal
error: [cli.main]   Error: Found extraneous } in Chalk template literal
    at tmp.replace (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/templates.js:109:11)
    at String.replace (<anonymous>)
    at module.exports (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/templates.js:99:6)
    at chalkTag (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/index.js:221:9)
    at Chalk.chalk.template (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/node_modules/chalk/index.js:36:20)
    at format (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/chalk-format.js:4:10)
    at chalkFormat (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/chalk-format.js:10:12)
    at ContentSection.arrayify.map.line (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/content-section.js:14:63)
    at Array.map (<anonymous>)
    at new ContentSection (/home/Jason/.nvm/versions/node/v9.11.1/lib/node_modules/polymer-cli/node_modules/command-line-usage/lib/content-section.js:14:51)
```

**Comment by jsilvermist** [comment]
Just noticed that `lint --help` is also broken.

**Comment by zeeshanhyder** [comment]
I got past the issue by following this post:
https://www.polymer-project.org/blog/2018-03-23-polymer-3-latest-preview

Instead of using npm, try installing dependencies with yarn. And make sure the version requirements for polymer and polymer-cli are correct.

**Comment by jsilvermist** [comment]
As of 1.7.7 `polymer serve --help` works, but `polymer lint --help` is still broken.

**Comment by zeeshanhyder** [comment]
@jsilvermist If you follow the guide it asks to install `polymer-cli@next`, which currently is **1.7.0-pre.17**. I tested in that and linter seems to be working fine there. If you can do install above version and try again.

**Comment by jsilvermist** [comment]
@zeeshanhyder `1.7.0-pre.17` is very old and should not be used, `polymer-cli@next` is also very old, and should not be used.

**Comment by 43081j** [comment]
this should be fixed by #612.

@aomarks would you mind closing this if you get a minute?
